internal/runners/show/show.go

```diff
@@ -217,14 +224,30 @@ func (s *Show) Run(params Params) error {
 		return locale.WrapError(err, "err_show_secrets", "Could not get secret information")
 	}
 
+	projectDir := filepath.Dir(s.project.Path())
+	if fileutils.IsSymlink(projectDir) {
+		projectDir, err = fileutils.SymlinkTarget(projectDir)
+		if err != nil {
+			return locale.WrapError(err, "err_show_projectdir", "Could not resolve project directory symlink")
+		}
+	}
+
 	rd := RuntimeDetails{
 		NameSpace:    fmt.Sprintf("%s/%s", owner, projectName),
 		Name:         projectName,
 		Organization: owner,
 		Visibility:   visibilityData(owner, projectName, remoteProject),
 		LastCommit:   commit,
+		Location:     projectDir,
 	}
 
+	projectTarget := target.NewProjectTarget(s.project, storage.CachePath(), nil, "").Dir()
+	if projectTarget != "" {
+		rd.Executables = setup.ExecDir(projectTarget)
+	}
+
+	fileutils.IsSymlink(filepath.Dir(s.project.Path()))
```

Comment:
I think you meant to either make this part of a conditional statement or omit it completely.

Reply:
Whoops, thanks for catching that!